examples/tutorials/audio_data_augmentation_tutorial.py

```diff
@@ -243,12 +238,24 @@ def get_noise_sample(*, resample=None):
 # audio. Doesn’t it sound more dramatic?
```

Comment:
(minor) imo this note would make a bit more sense after the displayed plots and audio output